servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -512,6 +513,7 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 	PagedAllocator<GeometryInstanceLightmapSH> geometry_instance_lightmap_sh;
 
 	void _geometry_instance_add_surface_with_material(GeometryInstanceForwardClustered *ginstance, uint32_t p_surface, SceneShaderForwardClustered::MaterialData *p_material, uint32_t p_material_id, uint32_t p_shader_id, RID p_mesh);
+	void _geometry_instance_add_surface_with_material_chain(GeometryInstanceForwardClustered *ginstance, uint32_t p_surface, SceneShaderForwardClustered::MaterialData *p_material, RID p_mat_src, RID p_mesh);
```

Comment:
I would prefer the naming "_geometry_instance_add_surface_with_material_next_pass". While chain is also a good name, the engine doesn't call ``next_pass`` materials "chains" anywhere, so the function name isn't entirely clear what it is doing.

Reply:
I was going to call it next_pass, but since I also moved the original call inside, it wasn't just the next pass. Is `_geometry_instance_add_surface_with_material_and_next_pass` acceptable?

Reply:
I think so. Let's see what reduz thinks as this was his idea in the first place